deploy.yaml

```diff
@@ -88,6 +68,22 @@
   roles:
     - role: databases/postgresql
 
+
+# Deploy InfluxDB to the environment
+# TODO: Add possibility of using an external InfluxDB by setting DB config variables
+# TODO: Support a HA cluster
+- hosts: dojot-k8s[0]
+  any_errors_fatal: true
+  tags:
+  - influxdb
+  - vernemq
+  - mosca
+  - lwm2m
+  - 100k
+  roles:
```

Comment:
If I pass the 100k and influxdb tags, will they both go up?

Reply:
Ansibile uses the Boolean logical operator `OR` to analyze the tags. So the answer to your question is yes! If the task have the tag "100k" or "influxdb" or both of then, it will be executed.